src/format.iced

```diff
@@ -0,0 +1,114 @@
+# This class prettifies the output of our armoring stream, according to the saltpack spec.
+# At the moment, per the spec, it simply frames the message, and inserts a space every 15 characters
+# and a newline every 200 words.
+
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+# Punctuation - this is modular
+space = new Buffer(' ')
+newline = new Buffer('\n')
+punctuation = new Buffer('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+# For working with the chunk streams
+noop = () ->
+
+# This stream takes basex'd input and writes framed output, as per https://saltpack.org/armoring#framing-the-basex-payload
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  # _format is the transform function passed to the chunk stream constructor
+  _format : (chunk) ->
+    # this is only ever called during flush()
+    if chunk.length < @block_size
+      return chunk
+
+    # write out the header if we haven't already
+    res = new Buffer('')
+    unless @_header_written
+      res = Buffer.concat([@_header, punctuation, space])
+      @_header_written = true
+
+    @_word_count++
+    if @_word_count isnt 200
+      # res will be an empty buffer except when writing the header
+      res = Buffer.concat([res, chunk, space])
+    else
+      # ditto
+      res = Buffer.concat([res, chunk, newline])
```

Comment:
I think you need to set `@_word_count` back to zero in this branch. Currently it looks like we're only printing the first newline, and then skipping the rest. Do you have a test vector that could've caught this? If not, it might be a good idea to add one.